src/hooks/navigation/useParticipantProjectsNav.tsx

```diff
@@ -0,0 +1,20 @@
+import { useCallback } from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { APP_PATHS } from 'src/constants';
+
+export default function useParticipantProjectsNav(projectId?: number) {
```

Comment:
The other hook was `useNavigateToParticipants` we could rename this and file to `useNavigateToParticipantProjects` or rename the other one. I have no preference except to have some convention.